src/include/stir/recon_buildblock/GibbsPrior.h

```diff
@@ -0,0 +1,170 @@
+/*
+    Copyright (C) 2000- 2011, Hammersmith Imanet Ltd
+    Copyright (C) 2025, University College London
+    Copyright (C) 2025, University of Milano-Bicocca
+    This file is part of STIR.
+
+    SPDX-License-Identifier: Apache-2.0
+
+    See STIR/LICENSE.txt for details.
+*/
+/*!
+  \file
+  \ingroup priors
+  \brief Declaration of the stir::GibbsPrior class
+
+  \author Kris Thielemans
+  \author Matteo Neel Colombo
+  \author Sanida Mustafovic
+
+*/
+
+#ifndef __stir_recon_buildblock_GibbsPrior_H__
+/**
+ * @file GibbsPrior.h
+ * \ingroup priors
```

Comment:
This doc should be moved down as class doc (remove the @file obviously, as you have one above already). That way, it will be transported to python etc (file doc isn't).

Reply:
Add the parsing parameters here, such that we don't need them in the derived classes